Sources/CartonHelpers/InteractiveWriter.swift

```diff
@@ -53,4 +53,12 @@ public final class InteractiveWriter {
       stream.flush()
     }
   }
+  
+  public func saveCursor() {
+    term?.write("\u{001B}[s")
+  }
+  
```

Comment:
<!--
  1 failure:  Lines should not ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/CartonHelpers/InteractiveWriter.swift;
  Line: 60;
-->
- :no_entry_sign: Lines should not have trailing whitespace. (`trailing_whitespace`)